assets/crd/0000_03_securityinternal-openshift_02_rangeallocation.crd.yaml

```diff
@@ -0,0 +1,51 @@
+# 0000_03_securityinternal-openshift_02_rangeallocation.crd.yaml
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    api-approved.openshift.io: https://github.com/openshift/api/pull/751
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+  name: rangeallocations.security.internal.openshift.io
```

Comment:
Would you more elaborate on why is this CRD needed?

Reply:
/assign @benluddy

Reply:
The namespace SCC allocation controller needs it, context here: https://github.com/openshift/api/pull/751. There's no cluster-config-controller in Microshift to provide it (https://github.com/openshift/cluster-config-operator/blob/0840c6f2f7f25ad9aba1a36597760f36ff0ab097/Dockerfile.rhel7#L13).